### ISSUE TITLE
Listen on specific interface address

### ISSUE THREAD

**Issue by alpharde** [NuSkooler/enigma-bbs#234]
Couldn't find this anywhere on the docs so I'm assuming we don't have such an option on the configuration files. I have a bunch of servers with multiple IP addresses and would like to specify one for enigma.

**Comment by NuSkooler** [comment]
My thinking is to expose an optional `address` member to the various servers in `config.hjson`. Example:
```
telnet: {
  port: 8888
  address: 10.0.2.5 // if missing, will use IPv6/IPv4 "any" address
}
```

Will that work for you?

**Comment by alpharde** [comment]
Yup, that would be nice.

**Comment by NuSkooler** [comment]
Latest [0.0.10-alpha](https://github.com/NuSkooler/enigma-bbs/tree/0.0.10-alpha) branch now has `address` that's set your `config.hjson` as described above for all services but NNTP. I'll get to adding it at some point later unless you need it right away.

**Comment by NuSkooler** [comment]
@alpharde did that give you what you need?

**Comment by NuSkooler** [comment]
Implemented in `0.0.10-alpha`, closing due to inactivity.
